Celeste.Mod.mm/Patches/CrystalStaticSpinner.cs

```diff
@@ -39,6 +78,85 @@ public patch_CrystalStaticSpinner(Vector2 position, bool attachToSolid, CrystalC
             orig_Awake(scene);
         }
 
+        public extern void orig_CreateSprites();
```

Comment:
`orig_CreateSprites()` isn't used at all. Mark the method as `MonoModReplace` instead.

I'd really prefer an IL hook or alternate solution to avoid redefining this method just to change the texture lookup.